### PR TITLE
Fixed tags for dependencies

### PR DESCRIPTION
**Pull request by RemDelaporteMathurin**
This PR adds fixed tags for dependencies.

This has the advantage of having more control over the dependencies changes + it's easier to retrigger the docker build/push workflow by simply changing the version of the dependency

@Shimwell what do you think?

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-paramak
-openmc-dagmc-wrapper
-openmc-plasma-source
-cad_to_h5m
+paramak=0.6.2
+openmc-dagmc-wrapper=0.2.2
+openmc-plasma-source=0.2.1
+cad_to_h5m=0.2.1
```
Comment (Shimwell) [inline requirements.txt:5]:
```suggestion
paramak==0.6.2
openmc-dagmc-wrapper==0.2.2
openmc-plasma-source==0.2.1
cad_to_h5m==0.2.1
```

Comment (Shimwell) [reply requirements.txt:5]: I think a == is needed instead of =.

Comment (RemDelaporteMathurin) [reply requirements.txt:5]: Good call

**Comment by RemDelaporteMathurin** [comment]
@Shimwell the tests don't seem to pass... any idea?

**Comment by Shimwell** [comment]
Looks like cubit is not finding the cubit learn lic file. I'm sure this has worked before so I guess we can look at other repos that use cubit and check fr differences

**Comment by RemDelaporteMathurin** [comment]
> Looks like cubit is not finding the cubit learn lic file. I'm sure this has worked before so I guess we can look at other repos that use cubit and check fr differences

Well the exact same error occured on neutronics_workflow https://github.com/fusion-energy/neutronics_workflow/runs/3729802402?check_suite_focus=true

**Comment by Shimwell** [comment]
it works over here https://github.com/Shimwell/cubit_plugin_testing

**Review by Shimwell — APPROVED** [review]

**Pull request merged**